src/openzaak/components/documenten/managers.py

```diff
@@ -0,0 +1,265 @@
+import logging
+from decimal import Decimal, InvalidOperation
+from privates.storages import private_media_storage
+
+from django.conf import settings
+from django.core.files import File
+from django.core.files.base import ContentFile
+from django.db.models import manager, fields
+
+from drc_cmis.client import CMISDRCClient, exceptions
+from drc_cmis.backend import CMISDRCStorageBackend
+
+from .query import InformatieobjectQuerySet
+
+logger = logging.getLogger(__name__)
+
+
+def convert_timestamp_to_django_date(json_date):
+    """
+    Takes an int such as 1467717221000 as input and returns 2016-07-05 as output.
+    """
+    if json_date is not None:
+        import datetime
+        timestamp = int(str(json_date)[:10])
+        django_date = datetime.datetime.fromtimestamp(timestamp).strftime('%Y-%m-%d')
+        return django_date
+
+
+def cmis_doc_to_django_model(cmis_doc):
+    from .models import EnkelvoudigInformatieObject, EnkelvoudigInformatieObjectCanonical
+
+    # The canonical field cannot be NULL
+    canonical = EnkelvoudigInformatieObjectCanonical.objects.create()
+
```

Comment:
Also, on the `canonical` object we should set `lock` if needed.

`canonical.lock = cmis_doc.locked  # or lock needs to be retrieved seperately from Alfresco?`

Reply:
Added canonical.lock = cmis_doc.versionSeriesCheckedOutId or ""

Reply:
I think this works, will review in depth details later.